ibis/backends/bigquery/tests/unit/snapshots/test_compiler/test_union/False/out.sql

```diff
@@ -1,7 +1,7 @@
 SELECT t0.`index`, t0.`Unnamed_0`, t0.`id`, t0.`bool_col`, t0.`tinyint_col`,
        t0.`smallint_col`, t0.`int_col`, t0.`bigint_col`, t0.`float_col`,
        t0.`double_col`, t0.`date_string_col`, t0.`string_col`,
-       t0.`timestamp_col`, t0.`year`, t0.`month`
+       t0.`datetime_col`, t0.`timestamp_col`, t0.`year`, t0.`month`
```

Comment:
It looks like you may have modified this table upstream in BigQuery.

I'd like to keep this schema fixed if possible. If you need to get codegen, it might be best to use a one-off table that doesn't require a table to exist in BigQuery.

You should be able to use unbound tables for this:

```python
t = ibis.table({"a": "timestamp", "b": "timestamp('UTC')"}, name="t")
# test codegen using `t`
```